compliance_checker/cf/util.py

```diff
@@ -294,6 +300,27 @@ def __iter__(self):
         return iter(itertools.chain(self._names, self._aliases))
 
 
+def download_cf_standard_name_table(version, location=None):
+    '''
+    Downloads the specified CF standard name table version and saves it to file
+
+    :param str version: CF standard name table version number (i.e 34)
+    :param str location: Path/filename to write downloaded xml file to
+    '''
+
+    if location is None:  # This case occurs when updating the packaged version from command line
+        location = 'compliance_checker/data/cf-standard-name-table.xml'
+
+    url = "http://cfconventions.org/Data/cf-standard-names/{0}/src/cf-standard-name-table.xml".format(version)
+    rhead = requests.head(url, allow_redirects=True)
+    if rhead.status_code == 200:
+        print("Downloading cf-standard-names table version {0} from: {1}".format(version, url))
+        urlretrieve(url, location)
```

Comment:
you use requests right above, why swithc to use urllib?  just use requests.  also don't make a head request, check its status and then continue - just `requests.get`